src/field_theory/galois_field.lean

```diff
@@ -0,0 +1,178 @@
+/-
+Copyright (c) 2021 Johan Commelin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Aaron Anderson, Alex J. Best, Johan Commelin, Eric Rodriguez, Ruben Van de Velde
+-/
+
+import algebra.char_p.algebra
+import field_theory.finite.basic
+import field_theory.separable
+import linear_algebra.finite_dimensional
+
+/-!
+# Galois fields
```

Comment:
Should this be in `field_theory/finite/`?

Reply:
I think that makes sense